external/PDF4QT/3rdparty_licenses/LittleCMS_COPYING.txt

```diff
@@ -0,0 +1,8 @@
+Little CMS
```

Comment:
that file shouldn't be necessary as we don't embed lcms source code. Same for other files in that directory

Reply:
Fixed, thanks